### DISCUSSION TITLE
Welcome to openwechat Discussions!

### DISCUSSION THREAD

**Discussion by eatmoreapple** [eatmoreapple/openwechat#89] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lihongcheng** [comment]
If you receive a voice message, how to parse it into a text message, can you give an example？
